assets/src/design-system/components/dropDown/select/components.js

```diff
@@ -47,10 +47,7 @@ export const SelectButton = styled.button(
     text-overflow: ellipsis;
     cursor: pointer;
 
-    ${themeHelpers.focusableOutlineCSS(
-      theme.colors.border.focus,
-      theme.colors.bg.secondary
-    )}
+    ${themeHelpers.focusableOutlineCSS}
```

Comment:
swapping this back to the default, we need to take a closer look at how to handle other color backgrounds but doing this here affects all of the dropDowns. I'm overriding the one in the animation panel separately below.